samples/bluetooth/peripheral_hids_mouse/src/main.c

```diff
@@ -106,17 +114,79 @@ static struct conn_mode {
 } conn_mode[CONFIG_BT_GATT_HIDS_MAX_CLIENT_COUNT];
 
 
-static void advertising_start(void)
+static void bond_find(const struct bt_bond_info *info, void *user_data)
+{
+	int err;
+
+	/* Filter already connected peers. */
+	for (size_t i = 0; i < CONFIG_BT_GATT_HIDS_MAX_CLIENT_COUNT; i++) {
+		if (conn_mode[i].conn) {
+			if (!memcmp(&info->addr,
```

Comment:
Could use bt_addr_cmp

Reply:
thanks, used bt_addr_le_cmp for type compatibility